Source/AST/Visitors/AttributedStringVisitor.swift

```diff
@@ -164,10 +169,29 @@ extension AttributedStringVisitor: Visitor {
     }
     
     public func visit(image node: Image) -> NSMutableAttributedString {
-        let s = visitChildren(of: node).joined
-        styler.style(image: s, title: node.title, url: node.url)
-        return s
-    }
+		let s = visitChildren(of: node).joined
+		#if os(iOS)		//Limited to iOS here due to CoreServices (UTType) and UIKit (NSTextAttachment) requirements
```

Comment:
Please move this new code into a private function to keep this function compact and easy to interpret